src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -39,7 +39,7 @@ def dialog_content
         VBox(
           alternative_information,
           create_alternatives_table,
-          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          RichText(Id(:slaves), "<pre>#{@alternative.choices.first.slaves}</pre>"),
```

Comment:
I do not get why it is only first choice slaves? If second choice have different slaves, then displaying what? or does it mean, that it have on start focus on first item, so always show first ones there?